CHANGELOG.md

```diff
@@ -17,7 +17,7 @@
 -----------
 
 ### Internals
-* None.
+* to_json API changed according to https://docs.google.com/document/d/1YtJN0sC89LMb4UVcPKFIfwC0Hsi9Vj7sIEP2vHQzVcY/edit?usp=sharing. Links to not embedded objects will never be followed.
```

Comment:
Do we usually use links to gdocs in the description of what has changed? Are those docs public?

Reply:
They are not public. But the this is under the "Internals" headline.